src/hooks/api/useSplits.ts

```diff
@@ -184,34 +197,125 @@ export function useAccountsTotal(
  */
 export function useAccountsMonthlyTotal(
   interval?: Interval,
-): UseQueryResult<AccountsMonthlyTotals> {
+): UseQueryResult<AccountsTotals[]> {
   interval = interval || Interval.fromDateTimes(
     DateTime.now().minus({ month: 6 }).startOf('month'),
     DateTime.now(),
   );
   const { data: accounts, dataUpdatedAt: accountsUpdatedAt } = useAccounts();
   const { data: prices, dataUpdatedAt: pricesUpdatedAt } = usePrices({});
 
+  const aggregate = React.useCallback(
+    ((data: AccountsTotals[]) => {
+      const dates = (interval as Interval).splitBy({ month: 1 }).map(d => (d.start as DateTime).endOf('month'));
+      dates[dates.length - 1] = (interval as Interval).end as DateTime;
```

Comment:
we set end of month for all dates because it's a monthly histogram so we want to see the accumulated monthly EXCEPT for the last one, where the selectedDate is important (as user may only want to see up until 15th of the month for example)